src/java/apiview-java-processor/src/main/java/com/azure/tools/apiview/processor/model/TypeKind.java

```diff
@@ -8,6 +8,10 @@ public enum TypeKind {
     CLASS("class"),
     INTERFACE("interface"),
     ENUM("enum"),
+    ANNOTATION("annotation"),
+    JAVA_MODULE("java_module"),
```

Comment:
I would just change this to be 'MODULE' with "module", and rename the css selectors appropriately.